.github/workflows/repo-health-job.yml

```diff
@@ -115,5 +120,5 @@ jobs:
           REPO_HEALTH_REPOS_WORKSHEET_ID: ${{ inputs.REPO_HEALTH_REPOS_WORKSHEET_ID }}
           REPO_HEALTH_OWNERSHIP_SPREADSHEET_URL: ${{ inputs.REPO_HEALTH_OWNERSHIP_SPREADSHEET_URL }}
         run: |
-          chmod +x edx-repo-health/scripts/repo-health-script.sh
```

Comment:
Both scripts' permissions have been updated on git level in `edx-repo-health` so we don't need to update the permissions on run time.